index.html

```diff
@@ -1,15 +1,341 @@
+                                alt="Hôtel le soleil du matin">
+                        </figure>
+                        <div class="populaire-info">
+                            <h4>Hôtel Le soleil du matin</h4>
+                            <p>Nuit à partir de 128€</p>
+
+                            <ul>
+
+                                <li><i class="fas fa-star"></i></li>
+                                <li><i class="fas fa-star"></i></li>
+                                <li><i class="fas fa-star"></i></li>
+                                <li><i class="fas fa-star"></i></li>
+                                <li><i class="fas fa-star"></i></li>
+                            </ul>
+                        </div>
+                    </a>
+                </article>
+
+                <article>
+                    <a href="#">
+
+                        <figure><img class="populaire-room"
+                                src="./assets/images/hebergements/4_small/aw-creative-VGs8z60yT2c-unsplash.jpg"
+                                alt=""></span>
+                            <div class="populaire-info">
+                                <h4>Au coeur de l'eau Chambre d'hôtes</h4>
+                                <p>Nuit à partir de 71€</p>
+
+                                <ul>
+
+                                    <li><i class="fas fa-star"></i></li>
+                                    <li><i class="fas fa-star"></i></li>
+                                    <li><i class="fas fa-star"></i></li>
+                                    <li><i class="fas fa-star"></i></li>
+                                    <li><i class="fas fa-star"></i></li>
+                                </ul>
+                            </div>
+                    </a>
+
+                </article>
+                <article>
+                    <a href="#">
+
+                        <figure><img class="populaire-room"
+                                src="./assets/images/hebergements/4_small/febrian-zakaria-sjvU0THccQA-unsplash.jpg"
+                                alt="">
+                        </figure>
+                        <div class="populaire-info">
+                            <h4>Hôtel Tout bleu et Blanc</h4>
+                            <p>Nuit à partir de 68€</p>
+
+                            <ul>
+                                <li><i class="fas fa-star"></i></li>
+                                <li><i class="fas fa-star"></i></li>
+                                <li><i class="fas fa-star"></i></li>
+                                <li><i class="fas fa-star"></i></li>
+                                <li><i class="fas fa-star"></i></li>
+
+                            </ul>
+                        </div>
+
+                    </a>
+                </article>
+        </section>
+<!--activities section-->
+        <section id="activity">
```

Comment:
id activities. 

Where is title of section ?

Reply:
indeed i missed this and just corrected it now